source/crud/bulk-write.md

```diff
@@ -90,6 +90,14 @@ class UpdateOneModel implements WriteModel {
      * value is false.
      */
     upsert: Optional<Boolean>;
+
+    /**
+     * Specify which document the operation updates if the query matches multiple
+     * documents. The first document matched by the sort order will be updated.
+     *
+     * This option is only sent if the caller explicitly provides a value.
```

Comment:
The corresponding model in `crud.md` also includes:

> This option is only supported by servers >= 8.0. Older servers will report an error for using this option.

Should that be included here?

The other line about a server-side error when used with `updateMany()` is rightfully omitted, since the UpdateOneModel and UpdateManyModel are distinguished here.

Reply:
This bulk write doc is for 8.0+ as described at: https://github.com/mongodb/specifications/blob/9479fa1c520aee2fb5a175c9f0973aabaa93b089/source/crud/bulk-write.md?plain=1#L4
So we don't need to include it for the sort field.